### DISCUSSION TITLE
Rename Tabs

### DISCUSSION THREAD

**Discussion by nullsweat** [vladmandic/sdnext#981] [Q&A] [answered]
Is there anyway to rename tabs? I would like to rename the tabs "From Text" back to txt2img, "From Image" to img2img, etc. I prefer the way it was in Auto1111, as goofy as I may be for asking this. I checked the config.json and the ui-config.json for anything that may resemble what I am looking for. In ui-config.json, I only found a couple entries similar to the following: 

"txt2img/Tabs@txt2img_extra_tabs/selected"

Not sure if this is what I would need to rename the tabs or if I'm even on the right track. I figured I'd ask here first before I break anything. Thanks!

**Comment by vladmandic** [comment, marked as answer]
No, sorry.
